### DISCUSSION TITLE
UP - DOWN - STOP per mqtt

### DISCUSSION THREAD

**Discussion by Kenaschon** [dewenni/ESP32-Jarolift-Controller#16] [General]
Eine Frage,

eine Rollade meldet sich per mqtt bei mir im ioBroker so.

mqtt.0.jarolift.Arbeitszimmer-Simone/status/shutter/1

Wenn sie sich bewegt ändert sich der Wert auf 0 oder auf 100.

Verbindung steht also.

Laut Anleitung soll die Ansteuerung per mqtt so aussehen:

mqtt.0.jarolift.Arbeitszimmer-Simone/cmd/shutter/1

Dort werden die Befehle "UP - DOWN - STOP" eingetragen.

Rollade bewegt sich aber nicht.

Was mache ich falsch?

**Comment by dewenni** [comment]
Hi,

ioBroker habe ich zum testen aktuell nicht laufen, aber generell ist es so:

- Das eigentliche Topic lautet bei dir wohl: `Arbeitszimmer-Simone`
- ein Kommando dazu lautet dann z.B. `Arbeitszimmer-Simone/cmd/shutter/1` mit Payload `up, open oder 0`

Im Log müsstest du dann sowas sehen:
```
[01.01.2025 - 10:24:15] I (APP-MQTT): msg received | topic: Arbeitszimmer-Simone/cmd/shutter/1 | payload: up
[01.01.2025 - 10:24:16] I (APP-JARO): execute cmd: UP - channel: 1
```

Im [MQTT-Explorer](https://mqtt-explorer.com) sieht das so aus:
<img width="202" alt="image" src="https://github.com/user-attachments/assets/aa1022eb-0244-4fcb-a7a1-829c9ce544f3" />

dort kannst du testweise auch Kommandos schreiben:
<img width="391" alt="image" src="https://github.com/user-attachments/assets/9960f726-ea86-4626-b18c-6ea5ca9bf688" />


Ich bin mir beim ioBroker nicht ganz sicher was da jetzt gesendet wird wenn du schreibst:
`mqtt.0.jarolift.Arbeitszimmer-Simone/cmd/shutter/1`

mqtt.0 ist ja nur der Adapter, aber ob Jarolift jetzt auch nicht in die Nachricht eingebaut wird, ist mir unklar.

**Comment by Kenaschon** [comment]
So sieht das bei mir aus.

![mqtt](https://github.com/user-attachments/assets/c3c08a1a-7fd0-4d3d-b890-49ad5edf6a0f)

**Reply by dewenni** [reply]
wie gesagt, installiere dir mal den matt-explorer falls noch nicht geschehen.
Damit kannst du das weiter analysieren.

Grundsätzlich sollte das mit deinen Namen passen. Ich habe es gerade nochmal kurz getestet.
<img width="1024" alt="image" src="https://github.com/user-attachments/assets/e723c0ba-dad4-4216-9102-d696cff21a81" />

Wenn es bei dir nicht klappt, dann liegt das wohl an irgendwelchen Einstellungen vom ioBroker.
Schau vielleicht auch nochmal hier. Das ist ja im Grunde das gleiche https://github.com/dewenni/ESP_Buderus_KM271/discussions/58#discussioncomment-7886204

**Comment by Kenaschon** [comment]
Danke für die Hilfe.

Habe jetzt alles noch einmal neu geflasht und eingestellt usw.. Nun läuft es wie es soll.

Keine Ahnung wo das Problem war.

Was mir noch aufgefallen ist.  Bei der Buderus-Steuerung ist es auf dem Handy schön passend zur Bildschirmbreite.

Bei dieser Steuerung geht es rechts an der Seite etwas raus. Kann das noch angepasst werden?

**Reply by dewenni** [reply]
ja das habe ich noch auf der ToDo Liste.
Werde ich demnächst mal noch anpassen.
